packages/ember-runtime/lib/system/array_proxy.js

```diff
@@ -33,6 +35,40 @@ var EMPTY = [];
 
 function K() { return this; }
 
+export function removeAt(array, start, len) {
+  if ('number' === typeof start) {
```

Comment:
it if requires it to be an array, there isn't a point to the standalone, there is only value migrating to extend prototypes being false by default if Ember isn't doing Ember.A() all over the place, which is very slow.